tools/run_build_bot.py

```diff
@@ -223,19 +223,21 @@ def test_request(merge_request):
 
     error_lines += test_output("nosetests -v", title="UNIT TESTS", test=lambda x: x.endswith("OK\n"))
     error_lines += test_output("flake8 .", title="FLAKE8")
-    error_lines += test_output('git diff HEAD^ HEAD|grep -P "^(?i)\+((.*#\s*NOQA:?\s*|(\s*#\s*flake8:\s*noqa\s*))$"',
-                               title="BROAD NOQA'S")
-    noqas = test_output('git diff HEAD^ HEAD|grep -P "^(?i)\+.*#\s*NOQA:\s*[a-z][0-9]{0,3}(\s*,\s*[a-z][0-9]{0,3})*$"',
-                        title="JUST NOQA'S")
+    noqas = test_output('git diff HEAD^ HEAD|grep -P "^(?i)\+((.*#\s*NOQA:?\s*|(\s*#\s*flake8:\s*noqa\s*))$"',
```

Comment:
Oh, then, ok. I'll do in a different way. Thank you for a comment.
I'd like to have my hands on testing it, but it's OK, I'll fix the code.